### PR TITLE
WIP: revise pivot_*() language  (closes #594)

### LINKED ISSUE #594: Revise language around pivot_longer and pivot_wider
The change to tidyr's `pivot_longer` and `pivot_wider` (from `gather` and `spread`, respectively) prompts a review of the language describing the use of these functions. The previous functions had verbs to describe their action (i.e. "gather" and "spread"), which were at least somewhat intuitive to understand. The verb for the new functions, "pivot", does not provide the same accessibility to a novice in understanding what is happening. After all, `pivot_longer` and `pivot_wider` are doing the opposite things to data, so using the same verb to describe their actions can be confusing. Consider revising the language of Episode 14, replacing the verb "pivot" with better description of what the two functions are actually doing.

### PR DESCRIPTION
**Pull request by stephhazlitt**
An attempt at adding some revised text around the `tidyr::pivot_longer()` and `pivot_wider()` functions, plus a few offered minor edits. This PR is a contribution to this [issue](https://github.com/swcarpentry/r-novice-gapminder/issues/594). I hope this is a start and helps the lead authors and community land on something helpful for novice learners.

### CODE REVIEW

File: _episodes_rmd/14-tidyr.Rmd [hunk 1]
```diff
@@ -3,14 +3,14 @@ title: Dataframe Manipulation with tidyr
 teaching: 30
 exercises: 15
 questions:
-- "How can I change the format of dataframes?"
+- "How can I change the layout of a dataframe?"
 objectives:
-- "To understand the concepts of 'long' and 'wide' data formats and be able
+- "To understand the concepts of 'longer' and 'wider' dataframe formats  and be able
```
Comment (jcoliver) [inline _episodes_rmd/14-tidyr.Rmd]: Is there an extra space on line 8, after "formats"?

File: _episodes_rmd/14-tidyr.Rmd [hunk 2]
```diff
@@ -212,10 +210,7 @@ gap_long$year <- as.integer(gap_long$year)
 
 ## From long to intermediate format with pivot_wider()
 
-It is always good to check work. So, let's use the opposite of `pivot_longer()` to
-'widen' our observation variables back out with the aptly named `pivot_wider()`. We
-can then pivot our `gap_long()` to the original intermediate format or the
-widest format. Let's start with the intermediate format.
+It is always good to check work. So, let's use the second `pivot` function, `pivot_wider()`, to 'widen' our observation variables back out.  `pivot_wider()` is the opposite of `pivot_longer()`, making a dataset wider by increasing the number of columns and decreasing the number of rows. We can use `pivot_wider()` to pivot or reshape our `gap_long()` to the original intermediate format or the widest format. Let's start with the intermediate format.
```
Comment (jcoliver) [inline _episodes_rmd/14-tidyr.Rmd]: I don't think `gap_long` is a function, so `gap_long()` should be replaced with `gap_long`.

**Comment by jcoliver** [comment]
Thanks, @stephhazlitt ! This is great. There are a couple of minor suggestions in the conversation above. Would you mind addressing them and updating the pull request?

**Comment by stephhazlitt** [comment]
Thanks for the review and catching those issues, both fixed and PR updated.

**Comment by jcoliver** [comment]
Thanks again, @stephhazlitt .

**Pull request merged**